pkg/cloudprovider/types.go

```diff
@@ -59,14 +58,14 @@ type InstanceType interface {
 	Offerings() []Offering
 	Architecture() string
 	OperatingSystems() sets.String
-	CPU() *resource.Quantity
-	Memory() *resource.Quantity
-	Pods() *resource.Quantity
-	NvidiaGPUs() *resource.Quantity
-	AMDGPUs() *resource.Quantity
-	AWSNeurons() *resource.Quantity
-	AWSPodENI() *resource.Quantity
+	// Resources are the full allocatable resource capacities for this instance type
+	Resources() v1.ResourceList
+	// Overhead is the amount of resource overhead expected to be used by kubelet and any other system daemons outside
+	// of Kubernetes.
 	Overhead() v1.ResourceList
+	// Price is a metric that is used by Karpenter's scheduler to prefer some instances over others.  This can be an
```

Comment:
I try to avoid using the name of a project inside a project. Not that we have any plans to change the name -- just as a best practice ;). 
```suggestion
	// Price is used to optimize pod placement onto nodes. This can be an
```